exchanges/btse/btse_websocket.go

```diff
@@ -27,6 +27,16 @@ const (
 	btseWebsocketTimer = time.Second * 57
 )
 
+var subscriptionNames = map[string]string{
+	subscription.MyTradesChannel:  "notificationApi",
+	subscription.AllTradesChannel: "tradeHistory",
+}
+
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.AllTradesChannel},
+	{Enabled: true, Channel: subscription.MyTradesChannel, Authenticated: true},
+}
+
```

Comment:
Please update`WsAuthenticate` `path` from `"/spotWS" + nonce` to `"/ws/spot" + nonce` so that auth endpoints can work

Reply:
Fixed [38c3ba59c](https://github.com/gbjk/gocryptotrader/commit/38c3ba59c)